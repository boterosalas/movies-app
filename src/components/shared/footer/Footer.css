.footer-container{
    width: 100%;
    padding: 8px 0px;
    color: #ffffff;
    text-align: center;
    font-size: 0.8rem;
}