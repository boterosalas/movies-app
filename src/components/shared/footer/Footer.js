import React from 'react';
import './Footer.css';

const Footer = () => {
    return (
        <footer className="footer-container">
            Â© MoviesApp - 2021
        </footer>
    )
}

export default Footer
