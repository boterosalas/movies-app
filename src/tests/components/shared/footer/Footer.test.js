import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import Footer from '../../../../components/shared/footer/Footer';


describe('Test on <Footer />',()=>{

    test('Should display the component correctly', () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('footer').text()).toContain('MoviesApp');
    });

    
})