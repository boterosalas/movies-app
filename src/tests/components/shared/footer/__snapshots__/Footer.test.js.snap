// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <Footer /> Should display the component correctly 1`] = `
<footer
  className="footer-container"
>
  Â© MoviesApp - 2021
</footer>
`;
